awscli/customizations/wizard/wizards/iam/new-role-wip.yml

```diff
@@ -214,7 +214,7 @@ execute:
         RoleName: "{role_name}"
         AssumeRolePolicyDocument: "{trust_policy}"
         Description: "{role_description}"
-      output_var: role_arn
```

Comment:
If we are going to rename the `role_arn` output variable, we will have to update it in the `write_config_file` step as well.